v2/thirdparty/common-customizations/saml/with-boxyhq/integration-steps.mdx

```diff
@@ -113,13 +184,7 @@ For our example app, you can see [this command here](https://github.com/supertok
 
 The output of this command will provide you the `client_id` and `client_secret` for this tenant. You will need to provide these values to SuperTokens for this tenant when configuring this tenant's providers (see below).
 
-:::info
-
-You can skip this step if you would be using dashboard for uploading the XML
-
-:::
-
-## 5) Create a new tenant in SuperTokens (if not done already)
+### 5) Create a new tenant in SuperTokens (if not done already)
```

Comment:
since this is via api, this should not have the dashboard code tab in it

Reply:
Done